examples/nlp/neural_machine_translation_with_keras_nlp.py

```diff
@@ -38,11 +38,14 @@
 """
 ## Setup
 
-Before we start implementing the pipeline, let's import all the libraries we need.
+Before we start implementing the pipeline, let's install and import all the
+libraries we need.
 """
 
 """shell
 pip install -q rouge-score
+pip uninstall -y keras-nlp
+pip install -q git+https://github.com/keras-team/keras-nlp.git@fc01b18125bb707a0de7e442a6a8e48c8fe913a3
```

Comment:
This should really be avoided -- we should install the latest version from pip.

Reply:
@fchollet, the reason for doing this is that certain APIs used in this example are not yet a part of any KerasNLP release. That's why I am using a particular commit for installation.

Reply:
I'd suggest so, yes.